canvas_modules/common-canvas/src/common-properties/controls/toggle/toggle.scss

```diff
@@ -14,10 +14,17 @@
  * limitations under the License.
  */
 
+.properties-toggle {
+	.properties-label-container {
+		margin-bottom: 0;
+		padding-bottom: 0;
+	}
+	.bx--toggle__switch {
+		margin-bottom: $spacing-05;
+	}
+}
 .properties-table-cell-control { // overrides when in a table cell
-	.properties-toggle {
-		.bx--toggle__switch { // override carbon when inside a table
-			margin: $spacing-02 0;
-		}
+	.bx--toggle__switch { // override carbon when inside a table
+		margin-bottom: $spacing-07;
```

Comment:
This doesn't seem right.  I think you're adding this because the label is there but it's being pushed off.  When in a table don't show the label from the control.